0400-office-data_management.Rmd

```diff
@@ -22,22 +22,23 @@ Exporting the PSCIS data to a gpkg is a great way to view everything spatially (
 
 To see the site photos, look in Mergin and/or Onedrive directories. 
 
-### PSCIS phase 1 data QA
+### PSCIS Phase 1 Data QA
 
 Here is a rough list of things to check at each site for phase 1 QA:
 
   + Correct spelling.
-  + Clear concise comments.
+  + Clear, concise comments.
   + Add any relevant info from climate change assessments comments or pictures to the main `assessment_comments` column.
   + Backwatered, is the box ticked, does it align with the photos?
   + Embeddedment, is the box ticked, does it align with the photos?
-  + Correct number of `moti_chris_culvert_id`. Check photos and comments to see how many culverts there are. If it's a major Moti structure than include the `chris_hwy_structure_road_id` instead. Beware that MoTi datasets are really bad and often incorrect.
   + Beaver activity, is the box ticked, does it align with the photos?
   + Fish observations during assessment, is the box ticked?
+  + Correct number of `moti_chris_culvert_id`. Check photos and comments to see how many culverts there are. If it's a major Moti structure than include the `chris_hwy_structure_road_id` instead. Beware that MoTi datasets are really bad and often incorrect.
```

Comment:
this sounds like there is only every 1 `moti_chris_culvert_id` but their can be up to three and their names are `moti_chris_culvert_id1`,`moti_chris_culvert_id2`, `moti_chris_culvert_id3`

> If it's a major Moti structure 

can define for clarity
(ie. major structures are structures with diameters or spans 3m or greater)